pangeo_forge/recipe.py

```diff
@@ -172,6 +172,11 @@ def _fsspec_safe_open(fname, **kwargs):
         with fp as fp2:
             yield fp2
 
+            
+def _get_url_size(fname):
+    with fsspec.open(fname, mode='rb') as of:
+        size = of.fs.size(fname)
```

Comment:
I think `of.size` should also exist as an attribute and potentially avoid a lookup